src/Process/PostDeploy/WarmUp.php

```diff
@@ -79,82 +72,49 @@ public function __construct(
      */
     public function execute()
     {
-        $client = $this->clientFactory->create();
-        $promises = [];
-
         try {
-            foreach ($this->getUrlsForWarmUp() as $url) {
-                $request = $this->requestFactory->create('GET', $url);
+            if (!$this->packageManager->has('magento/magento-cloud-components', '*')) {
```

Comment:
It is required, why do we need this check then?

Reply:
Done